exercises/visibility_user_data/visibility-user-data

```diff
@@ -0,0 +1,83 @@
+#cloud-config
```

Comment:
The directory name should be user-data and the file name should be visibility_user_data. We need to change this.